### PR TITLE
Onboarding V2 Privacy settings view

### LINKED ISSUE #12196: Set advanced Privacy Settings View
Screen Shot 2021-09-28 at 12 02 10 PM

### PR DESCRIPTION
**Pull request by adonesky1**
Fixes: #12196
Adds privacy-settings view as part of [onboarding v2](https://www.figma.com/file/iCOjL3k1BkQqpYHFUVVFUZ/Extension-Onboarding?node-id=1525%3A15924) implementation

Screenshot:
Screen Shot 2021-09-30 at 5 27 29 PM

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]

**Review by chingon81 — APPROVED** [review]

File: ui/pages/onboarding-flow/privacy-settings/privacy-settings.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+  const handleSubmit = () => {
+    dispatch(
+      setFeatureFlag('showIncomingTransactions', showIncomingTransactions),
+    );
+    dispatch(setUsePhishDetect(usePhishingDetection));
+    dispatch(setUseTokenDetection(turnOnTokenDetection));
+    history.push(ONBOARDING_PIN_EXTENSION_ROUTE);
+  };
+
```
Comment (jpuri) [inline ui/pages/onboarding-flow/privacy-settings/privacy-settings.js:39]: `renderSetting` can be extracted into small component of its own.

Comment (adonesky1) [reply ui/pages/onboarding-flow/privacy-settings/privacy-settings.js:39]: Sure will do!

Comment (adonesky1) [reply ui/pages/onboarding-flow/privacy-settings/privacy-settings.js:38]: Extracted @jpuri 😄

File: ui/pages/onboarding-flow/privacy-settings/index.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  &__settings {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    max-width: 620px;
+    margin-bottom: 20px;
+
+    & a {
```
Comment (darkwing) [inline ui/pages/onboarding-flow/privacy-settings/index.scss:24]: Do we need `&` here?

**Review by darkwing — DISMISSED** [review]

File: ui/pages/onboarding-flow/privacy-settings/index.scss (hunk 2 above)
Comment (adonesky1) [reply ui/pages/onboarding-flow/privacy-settings/index.scss]: apparently not. Removed @darkwing !

**Review by ryanml — DISMISSED** [review]
lgtm!

**Review by darkwing — DISMISSED** [review]

**Review by darkwing — APPROVED** [review]

**Review by ryanml — APPROVED** [review]

**Pull request merged**